### PR TITLE
Signup login

### PR DESCRIPTION
**Pull request by mohammed-sas**
[preview](https://deploy-preview-2--mhd-video-library.netlify.app/)

### Features 
1.added signup functionality
2.added login functionality
3.added router for the app
4.added routing in the navbar

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]
Functionality is working as expected

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
 import "./App.css";
 import Navbar from "./components/navbar/Navbar";
-
+import { Routes,Route } from "react-router";
 import Home from "./pages/home/Home";
+import Login from "./pages/login/Login";
+import Signup from "./pages/signup/Signup";
```
Comment (AnkurChunekar) [inline src/App.js:6]: you can create an index.js file in pages, and import all of the above from a single file

**Comment by AnkurChunekar** [comment]
screen-recorder-thu-mar-24-2022-16-09-24

Form validation: user is allowed to sign in even if password and confirm password values do not match

**Review by AnkurChunekar — APPROVED** [review]
Rest everything looks good, 
- code is very neat and clean, 
- files are structured properly
- great work @mohammed-sas

File: src/App.js (hunk 1 above)
Comment (mohammed-sas) [reply src/App.js:6]: okay

**Pull request merged**